src/app/shared/components/template/components/dashed-box/dashed-box.component.ts

```diff
@@ -3,7 +3,7 @@ import { TemplateBaseComponent } from "../base";
 import { ITemplateRowProps } from "../../models";
 import { getStringParamFromTemplateRow } from "../../../../utils";
 import { DomSanitizer, SafeHtml } from "@angular/platform-browser";
-import marked from "marked";
+import { parseMarkdown } from "../../../../utils";
```

Comment:
nit(non-blocking) you've used relative import here but absolute on file below, might be good to try be more consistent. Also import can be merged with the `getStringParamFromTemplateRow`